doc/install_mne_c.rst

```diff
@@ -83,53 +83,79 @@ Installing from a Mac OSX disk image
 Setting up MNE-C environment
 ############################
 
-The system-dependent location of the MNE Software will be
-here referred to by the environment variable MNE_ROOT. There are
-two scripts for setting up user environment so that the software
-can be used conveniently:
+Your system-dependent path to the MNE Software will be referred to by the environment variable MNE_ROOT. 
+
+Your system-dependent path to MATLAB Software will be referred to by the environment variable MATLAB_ROOT.
+
+You can download a free runtime distribution of Matlab with the libraries necessary to run Matlab code from,
+https://www.mathworks.com/products/compiler/matlab-runtime.html
+
+If you do not have a Matlab distribution, then leave MATLAB_ROOT undefined. Please note however that if Matlab
+is not available, the utilities mne_convert_mne_data, mne_epochs2mat, mne_raw2mat, and mne_simu will not work.
+
+To setup the MNE environment, a script must be sourced into your current shell.
+The result will be that additional environment varaibles like MNE_LIB_PATH will be set in your current shell 
+in order to run the MNE Software.
+
+A note about source-ing scripts with "source" and "." (dot). The hash bang at the top of a script tells 
+exec what shell to use by default to interpret the script. But the hash bang line is ignored when you 
+source a script via either the builtin "source" or "." directive.  So there needs to be a setup script
+compatible with your current shell, i.e., if you use csh you cannot source a bash script.  (BTW, the "." (dot)
+directive will search for the script in $PATH if there is no slash in the script argument while "source" 
+does not search $PATH).
+
+For Bourne or bash compatible shells, e.g., sh/bash/zsh, the script to source is:
 
 .. code-block:: console
 
-    $ . $MNE_ROOT/bin/mne_setup_sh
+    $ $MNE_ROOT/bin/mne_setup_sh
```

Comment:
actually I would not put this in console mode as we just refer to the file here and don't suggest to type this in a terminal. Maybe using backticks is enough